pandas/core/internals.py

```diff
@@ -4066,6 +4076,27 @@ def canonicalize(block):
         return all(block.equals(oblock)
                    for block, oblock in zip(self_blocks, other_blocks))
 
+    def unstack(self, unstacker):
+        """Return blockmanager with all blocks unstacked"""
```

Comment:
can you expand the doc string a bit (IOW document what unstacker is supposed to be)